src/Printer.php

```diff
@@ -41,21 +41,21 @@ public function print(string $json, string $indent = '    ', string $newLine = \
         if (null === \json_decode($json) && \JSON_ERROR_NONE !== \json_last_error()) {
             throw new \InvalidArgumentException(\sprintf(
                 '"%s" is not valid JSON.',
-                $json,
```

Comment:
@piotrszymanski81 

This is valid syntax on PHP 7.3.

💁‍♂️ For reference, see https://www.php.net/manual/en/migration73.new-features.php#migration73.new-features.core.trailing-commas.

Reply:
in fact, i found another bug in my design
thanks for the verification

Reply:
You are welcome, @piotrszymanski81! Thank you for taking the time to open an issue!